ddl/backfilling_scheduler.go

```diff
@@ -262,24 +266,31 @@ type ingestBackfillScheduler struct {
 	resultCh chan *backfillResult
 
 	copReqSenderPool *copReqSenderPool
+	taskMaxID        int
 
 	writerPool  *workerpool.WorkerPool[idxRecResult]
 	writerMaxID int
 	poolErr     chan error
 	backendCtx  *ingest.BackendContext
+
+	checkpointMgr ingest.CheckpointManager
 }
 
-func newIngestBackfillScheduler(ctx context.Context, info *reorgInfo, tbl table.PhysicalTable) *ingestBackfillScheduler {
+func newIngestBackfillScheduler(ctx context.Context, info *reorgInfo,
+	sessPool *sess.Pool, tbl table.PhysicalTable) *ingestBackfillScheduler {
 	return &ingestBackfillScheduler{
 		ctx:       ctx,
 		reorgInfo: info,
+		sessPool:  sessPool,
 		tbl:       tbl,
 		taskCh:    make(chan *reorgBackfillTask, backfillTaskChanSize),
 		resultCh:  make(chan *backfillResult, backfillTaskChanSize),
 		poolErr:   make(chan error),
 	}
 }
 
+const checkpointUpdateInterval = 5 * time.Second
```

Comment:
Since it is a checkpoint update interval, should we set it longer? for example 10 minutes?  it seems to redo 10 mins will not cause a very big impact on the user.

Reply:
This is used for integration test. I think it is reasonable to be 10 min.

Reply:
what I mean is checkpoint could be a little bit bigger time period.